Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -29,6 +29,8 @@ public class NearInteractionTouchable : ColliderNearInteractionTouchable
         protected Vector3 localUp = Vector3.up;
         public Vector3 LocalUp { get => localUp; }
 
+        public bool AreLocalVectorsOrthogonal => Vector3.Dot(localForward, localUp) == 0;
+
```

Comment:
Nit: Add a doccomment (in particular, it would be good to clarify that it only reasons over LocalForward and LocalUp - there are other Local* vectors which aren't involved in this calculation and I might expect as a reader to see those somehow factored in)

Reply:
done